ansible/roles/geotrellis.libgdal-java/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Update the .bashrc to set JAVA_GDAL_DIR so that GeoTrellis can find GDAL bindings
+  lineinfile:
+    dest=/home/vagrant/.bashrc
```

Comment:
Does it make sense to use `/etc/environment` for this vs. `.bashrc`? In the context of this project, I could see `.bashrc` being a valid option, but if we had to break this out, I think `/etc/environment` would make more sense.